studio/components/interfaces/Organization/Documents/DPA.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  ScaffoldSection,
+  ScaffoldSectionContent,
+  ScaffoldSectionDetail,
+} from 'components/layouts/Scaffold'
+import { Button, IconExternalLink } from 'ui'
+
+const DPA = () => {
+  return (
+    <>
+      <ScaffoldSection>
+        <ScaffoldSectionDetail>
+          <div className="sticky space-y-6 top-12">
+            <p className="text-base m-0">Data Processing Addendum (DPA)</p>
```

Comment:
there's possibly a `<ScaffoldDetailTitle>` component

Reply:
Sorry couldn't find this. I tried` <ScaffoldTitle>` and `<ScaffoldHeader>` but it looks inconsistent with the other tabs. Maybe help suggest a change if you really need it as a different component?